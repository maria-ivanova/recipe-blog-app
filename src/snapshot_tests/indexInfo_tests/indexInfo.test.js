import React from 'react';
import renderer from 'react-test-renderer';
import Routing from '../helpers/router.js';
import IndexInfo from '../../components/indexInfo.js';

describe('Index Info Component', () => {
    it('should render layout', () => {
        const tree = renderer.create(
            <Routing>
                <IndexInfo/>
            </Routing>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})