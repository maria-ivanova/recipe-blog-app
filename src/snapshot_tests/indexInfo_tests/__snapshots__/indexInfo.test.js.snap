// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Info Component should render layout 1`] = `
<section
  className="sec content_sec bg_sec"
>
  <div
    className="container"
  >
    <h2
      className="page_title"
    >
      <span>
        
      </span>
    </h2>
    <div
      className="inner_txt"
    >
      На нашия уебсайт можете да намерите селекции от рецепти, групирани по категории, 
      <br />
      написани от хиляди кулинари от всички краища на страната!
    </div>
    <div
      className="cells_box"
    >
      <span
        className="cell"
      >
        <img
          alt="soup"
          src="i_soup.svg"
        />
        <span
          className="title"
        >
          Салати
        </span>
      </span>
      <span
        className="cell"
      >
        <img
          alt="dish"
          src="i_mainDish.svg"
        />
        <span
          className="title"
        >
          Основни ястия
        </span>
      </span>
      <span
        className="cell"
      >
        <img
          alt="soup"
          src="i_soup.svg"
        />
        <span
          className="title"
        >
          Супи
        </span>
      </span>
      <span
        className="cell"
      >
        <img
          alt="dessert"
          src="i_dessert.svg"
        />
        <span
          className="title"
        >
          Десерти
        </span>
      </span>
    </div>
  </div>
</section>
`;
